server/bleep/src/webserver.rs

```diff
@@ -96,22 +96,20 @@ pub async fn start(app: Application) -> Result<()> {
 
     let mut router = Router::new().nest("/api", api);
 
-    if let Environment::PrivateServer = app.env {
```

Comment:
is the check for `app.env == PrivateServer` not required anymore here?

Reply:
if it is, please derive `PartialEq`and use `if .. == ..` instead of `if let .. = ..`

Reply:
I removed this capability, because I don't think the separation here makes sense. The flag itself controls whether this feature is on or off.